cypress/integration/settings/finance/expense-classes-create.spec.js

```diff
@@ -0,0 +1,28 @@
+import NewExpenceClass from '../../../support/fragments/settings/finance/newExpenseClass';
+import TestType from '../../../support/dictionary/testTypes';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import devTeams from '../../../support/dictionary/devTeams';
+import SettingsFinance from '../../../support/fragments/settings/finance/settingsFinance';
+
+describe('ui-invoices-settings: Batch Group creation', () => {
+  const expenseClass = { ...NewExpenceClass.defaultUiBatchGroup };
+  const newExpenseClass = { ...NewExpenceClass.defaultUiBatchGroup };
+  before(() => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
```

Comment:
I think it's better use cy.loginAsAdmin()

Reply:
Done